addons/mail/models/discuss/discuss_channel.py

```diff
@@ -912,15 +918,9 @@ def _to_store(self, store: Store):
                 info['message_needaction_counter'] = channel.message_needaction_counter
                 info["message_needaction_counter_bus_id"] = bus_last_id
                 if member:
-                    member_data = list(
-                        member._discuss_channel_member_format(
-                            extra_fields={"last_interest_dt": True, "new_message_separator": True}
-                        ).values()
-                    )
+                    member_data = [self_member_data]
```

Comment:
I'd like to wait on https://github.com/odoo/odoo/pull/170330 to merge this to avoid conflict